features/order/managing_orders/cancelling_order.feature

```diff
@@ -28,3 +28,10 @@ Feature: Cancelling orders
         Given the customer canceled this order
         When I view the summary of the order "#00000022"
         Then I should not be able to cancel this order
+
+    @ui
+    Scenario: Checking order payment state of a cancelled order
+        Given I view the summary of the order "#00000022"
+        And I cancel this order
```

Comment:
The `Given the customer canceled this order` step should be renamed to the above. The implementation is fine :)